tests/Executor/DeferredFieldsTest.php

```diff
@@ -529,14 +530,14 @@ public function testComplexRecursiveDeferredFields() : void
             ['deferredNest', 'nest', 'deferred'],
             ['deferredNest', 'deferredNest'],
 
-            ['!dfd for: ', ['nest', 'deferredNest', 'deferred']],
-            ['!dfd for: ', ['deferredNest', 'deferred']],
-            ['!dfd for: ', ['deferredNest', 'nest', 'deferred']],
-            ['!dfd nest for: ', ['deferredNest', 'deferredNest']],
+            ['!dfd for: ', 'nest', 'deferredNest', 'deferred'],
+            ['!dfd for: ', 'deferredNest', 'deferred'],
+            ['!dfd for: ', 'deferredNest', 'nest', 'deferred'],
+            ['!dfd nest for: ', 'deferredNest', 'deferredNest'],
 
             ['deferredNest', 'deferredNest', 'sync'],
             ['deferredNest', 'deferredNest', 'deferred'],
-            ['!dfd for: ', ['deferredNest', 'deferredNest', 'deferred']],
```

Comment:
This whole file change seems wrong. Nesting here is important for sane maintenance. We should bail out of typing tests or maybe just set the type of path as `array<mixed>`. Please revert this.

Reply:
Yeah, I must have done that late at night. reverted to what is on master in 18ed4af